config/config.py

```diff
@@ -15,31 +17,13 @@
     "澳门": "Macao",
 }
 
-country_map = {
-    "泰国": "Thailand",
-    "日本": "Japan",
-    "新加坡": "Singapore",
-    "韩国": "South Korea",
-    "澳大利亚": "Australia",
-    "马来西亚": "Malaysia",
-    "美国": "US",
-    "法国": "France",
-    "德国": "Germany",
-    "阿联酋": "Arab Emirates",
-    "加拿大": "Canada",
-    "越南": "Vietnam",
-    "意大利": "Italy",
-    "英国": "UK",
-    "俄罗斯": "Russia",
-    "芬兰": "Finland",
-    "西班牙": "Spain",
-    "瑞典": "Sweden",
-    "印度": "India",
-    "斯里兰卡": "Sri Lanka",
-    "柬埔寨": "Cambodia",
-    "尼泊尔": "Nepal",
-    "菲律宾": "Philippines",
-    "钻石公主号邮轮": "Japan (Diamond Princess)",
-    "埃及": "Egypt",
-    "比利时": "Belgium"
-}
+
+def load_country_map():
+    countries_en = dict(countries_for_language('en'))
+    countries_cn = dict(countries_for_language('zh_Hans_CN'))
+    country_map = dict()
+    for unicode, country in countries_cn.items():
+        if unicode != 'CN':

```

Comment:
I have noticed the website includes Macao, Hong Kong, and Taiwan as part of China. However, in the code they are **not** included and thus, are calculated separately.

For this reason, I have excluded China from the map. Its statistics are calculated as previously (without Macao, Hong Kong etc).